packages/orbit-components/src/Card/__tests__/index.test.tsx

```diff
@@ -22,6 +23,12 @@ describe("Card", () => {
     expect(screen.getByText("description")).toBeInTheDocument();
     expect(screen.getByTestId("airplane"));
     expect(screen.getByRole("button"));
+    expect(screen.getByTestId("test")).toHaveStyle({
+      marginTop: "12px",
+      marginBottom: "0",
+      marginRight: "0",
+      marginLeft: "0",
+    });
```

Comment:
nope. It should be tested here too because it's also based on space after property. 
Isolated testing of the utility function is not guaranteed, that property is passed down to a styled component and will be applied. So it's ok in that case to have that test. We were testing spaceAfter before and do not think we should change it here.

Reply:
I am not saying we should not test. We should. But the assertion shouldn't be the result of the function. Even with the spaceAfter prop we should not break the abstraction barrier. The components should trust that the function returns what is expected, because the function should be tested in its place. And therefore the test should be that the function is called when it should be called.

Also, what should be the expected behavior when one uses `margin` and `spaceAfter`? From reading the code, it seems that the `margin` prop is ignored. I don't think it should be like that. Also, there is no test for such scenario, and there should be, once it is correctly defined what should happen.